diff --git a/dist/presets/netlify/utils.mjs b/dist/presets/netlify/utils.mjs
index 74ae91595acf3fad9cf3ac9bad171f8ee6f1f317..7a87bdb4f3d33501f73cf31fbe3c2f615664076c 100644
--- a/dist/presets/netlify/utils.mjs
+++ b/dist/presets/netlify/utils.mjs
@@ -93,6 +93,8 @@ export const config = {
   name: "server handler",
   generator: "${getGeneratorString(nitro)}",
   path: "/*",
+  nodeBundler: "none",
+  includedFiles: ["**"],
   excludedPath: ${JSON.stringify(getStaticPaths(nitro.options.publicAssets))},
   preferStatic: true,
 };
