diff --git a/dist/presets/netlify/utils.mjs b/dist/presets/netlify/utils.mjs
index 74ae91595acf3fad9cf3ac9bad171f8ee6f1f317..aed8f17e6431a49e6f20faabc374fcc5664eee71 100644
--- a/dist/presets/netlify/utils.mjs
+++ b/dist/presets/netlify/utils.mjs
@@ -93,6 +93,7 @@ export const config = {
   name: "server handler",
   generator: "${getGeneratorString(nitro)}",
   path: "/*",
+  nodeModuleFormat: "esm",
   excludedPath: ${JSON.stringify(getStaticPaths(nitro.options.publicAssets))},
   preferStatic: true,
 };
